python/lsst/ts/MTMount/mtmount_csc.py

```diff
@@ -248,30 +247,22 @@ async def begin_enable(self, data):
                     f"The CSC was not allowed to command the mount: {e!r}"
                 )
 
-        self.enable_task.cancel()
         self.enable_task = asyncio.create_task(self.enable_devices())
```

Comment:
enable_task is assigned meaningful value only here. Wouldn't it be better to drop the self.enable_task variable and simply await self.enable_devices() (instead of await self.enabled_task)?

Reply:
Saving the task this allows me to cancel it elsewhere (e.g. when changing out of enabled state). I agree it is an ugly paradigm but I have not figured a way around it.